osu.Game/Overlays/Mods/ModPresetRow.cs

```diff
@@ -39,26 +48,58 @@ public ModPresetRow(Mod mod)
                         },
                         new OsuSpriteText
                         {
-                            Text = mod.Name,
-                            Font = OsuFont.Default.With(size: 16, weight: FontWeight.SemiBold),
-                            Origin = Anchor.CentreLeft,
                             Anchor = Anchor.CentreLeft,
-                            Margin = new MarginPadding { Bottom = 2 }
-                        }
+                            Origin = Anchor.CentreLeft,
+                            Font = OsuFont.Torus.With(size: 16f, weight: FontWeight.SemiBold),
+                            Colour = colourProvider.Content1,
+                            UseFullGlyphHeight = false,
+                            Text = mod.Name,
+                        },
                     }
-                }
-            };
-
-            if (!string.IsNullOrEmpty(mod.SettingDescription))
-            {
-                AddInternal(new OsuTextFlowContainer
+                },
+                new Container
                 {
                     RelativeSizeAxes = Axes.X,
                     AutoSizeAxes = Axes.Y,
-                    Padding = new MarginPadding { Left = 14 },
-                    Text = mod.SettingDescription
-                });
-            }
+                    Padding = new MarginPadding { Horizontal = 10f },
+                    Alpha = mod.SettingDescription.Any() ? 1 : 0,
+                    Children = new Drawable[]
+                    {
+                        new TextFlowContainer(t =>
+                        {
+                            t.Font = OsuFont.Torus.With(size: 12f, weight: FontWeight.SemiBold);
+                        })
+                        {
+                            AutoSizeAxes = Axes.Both,
+                            Colour = colourProvider.Content2,
+                            Text = string.Join('\n', mod.SettingDescription.Select(svp => svp.setting)),
+                        },
+                        new TextFlowContainer(t =>
+                        {
+                            t.Font = OsuFont.Torus.With(size: 12f, weight: FontWeight.SemiBold);
+                        })
+                        {
+                            Anchor = Anchor.TopRight,
+                            Origin = Anchor.TopRight,
```

Comment:
I have grave concerns about this breaking in the future and requiring proper grid layout. But I'm going to go with this and we'll see if you have to rewrite this in a day or two.